include/cosim/osp_config_parser.hpp

```diff
@@ -24,6 +25,9 @@ struct osp_config
     /// The system structure
     cosim::system_structure system_structure;
 
+    // The algorithm
+    std::variant<cosim::fixed_step_configuration, cosim::ecco_parameters> algorithm_configuration;
```

Comment:
Now we have this, can we remove the `step_size` field further down?

Reply:
Yes. :D